common/libs/values.js

```diff
@@ -0,0 +1,23 @@
+// @flow
+import type Big from 'big.js';
+
+export function padLeft(n: string, width: number, z: string = '0'): string {
+  return n.length >= width ? n : new Array(width - n.length + 1).join(z) + n;
+}
+
+// FIXME does this even makes sense
+export function toHex(n: Big): string {
```

Comment:
is this actually toHex? If so, is there no unit-tested library out there that can do this?